package.json

```diff
@@ -50,19 +50,18 @@
     "@types/jest": "27.4.1",
     "@types/k6": "0.36.0",
     "@types/node": "16.11.26",
-    "@typescript-eslint/eslint-plugin": "4.33.0",
-    "@typescript-eslint/parser": "4.33.0",
+    "@typescript-eslint/eslint-plugin": "5.27.0",
+    "@typescript-eslint/parser": "5.27.0",
     "apollo-server": "3.5.0",
     "benchmark": "2.1.4",
     "bob-the-bundler": "1.6.1",
     "chalk": "5.0.1",
-    "eslint": "7.32.0",
+    "eslint": "8.17.0",
     "eslint-config-prettier": "8.5.0",
-    "eslint-config-standard": "16.0.3",
-    "eslint-plugin-import": "2.25.4",
-    "eslint-plugin-node": "11.1.0",
+    "eslint-config-standard": "17.0.0",
+    "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-n": "^15.2.1",
```

Comment:
`eslint-plugin-node` is unmaintened, `eslint-config-standard` now uses under the hood `eslint-plugin-n`